src/private/meeting.ts

```diff
@@ -0,0 +1,168 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError } from '../public/interfaces';
+
+export namespace meeting {
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent a meeting room.
+   */
+  export interface MeetingRoomInfo {
+    /**
+     * Endpoint id of the meeting room.
+     */
+    endpointId: string;
+    /**
+     * Device name of the meeting room.
+     */
+    deviceName: string;
+    /**
+     * Client type of the meeting room.
+     */
+    clientType: string;
+    /**
+     * Client version of the meeting room.
+     */
+    clientVersion: string;
+  }
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent capabilities of a meeting room.
+   */
+  export interface MeetingRoomCapability {
+    /**
+     * Media control capabilities, value can be "toggleMute", "toggleCamera", "toggleCaptions", "volume".
+     */
+    mediaControls: string[];
+    /**
+     * Main stage layout control capabilities, value can be "showVideoGallery", "showContent", "showVideoGalleryAndContent", "showLargeGallery", "showTogether".
+     */
+    stageLayoutControls: string[];
+    /**
+     * Meeting control capabilities, value can be "leaveMeeting".
+     */
+    meetingControls: string[];
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to represent states of a meeting room.
+   */
+  export interface MeetingRoomState {
+    /**
+     * Current mute state, true: mute, false: unmute.
+     */
+    toggleMute: boolean;
+    /**
+     * Current camera state, true: camera on, false: camera off.
+     */
+    toggleCamera: boolean;
+    /**
+     * Current captions state, true: captions on, false: captions off.
+     */
+    toggleCaptions: boolean;
+    /**
+     * Current main stage layout state, value can be one of "Gallery", "Content + gallery", "Content", "Large gallery" and "Together mode".
+     */
+    stageLayout: string;
+    /**
+     * Current leaveMeeting state, true: leave, false: no-op.
+     */
+    leaveMeeting: boolean;
+  }
+
+  let meetingRoomCapabilitiesUpdateHandler: (capabilities: MeetingRoomCapability) => void;
+  GlobalVars.handlers['meeting.meetingRoomCapabilitiesUpdate'] = handleMeetingRoomCapabilitiesUpdate;
+  let meetingRoomStatesUpdateHandler: (states: MeetingRoomState) => void;
```

Comment:
These kind of handler are generally define indie GlobalVars file

Reply:
Moved two handler variables to GlobalVars.